.golangci.yml

```diff
@@ -1,9 +1,7 @@
 ---
-exclude-dirs:
-  - vendor
 linters:
   enable:
   - bodyclose
-run:
-  skip-dirs:
+issues:
+  exclude-dirs:
     - vendor
```

Comment:
I removed this in completely since `vendor/` is gone, not big deal if this comes first I can rebase and remove again:

https://github.com/osbuild/image-builder-crc/pull/1476

Reply:
Apologies, the other PR got merged, please `rebase -Xtheirs main`, cheers.